commands/activation.go

```diff
@@ -25,7 +25,7 @@ import (
     "time"
 
     "github.com/openwhisk/openwhisk-client-go/whisk"
-    "github.com/openwhisk/openwhisk-cli/wski18n"
+    "../wski18n"
```

Comment:
Alright, after running `$ go get github.com/mitchellh/go-homedir`, `go build -o wsk` produces wsk CLI successfully.

Finally, with this PR I am able to run `gradle distDocker` and `go get github.com/mitchellh/go-homedir && go build -o wsk` with success.